DCode.Services/Reporting/ReportingService.cs

```diff
@@ -90,9 +90,31 @@ public DatabaseTable ExecuteDbQuery(string query)
             return _dbQueryManager.RunQuery(query);
         }
 
-        public IEnumerable<string> GetAllActiveUsers()
+        public IEnumerable<string> GetConsultingUsers()
         {
-            return _userRepository.GetAllActiveUsers();
+            return new List<string>
+            {
+                "USMUMSIALL@deloitte.com",

```

Comment:
Hi @ritwiksen ,
Lets pick these as semicolon separated string from config file. I am not sure if it will be a good idea to keep these in code file.

Let me know your view.